roles/base/tasks/main.yml

```diff
@@ -18,6 +39,7 @@
     - librbd-dev
     - lshw
   when: ansible_os_family == "Debian"
+  ignore_errors: yes
```

Comment:
This task is absolutely critical; it installs the dependencies needed by 
several parts of our project.

If it's erroring consistently, we can probably work around that, we just 
need to know what's happening.

-Erik

On 6 Dec 2015, at 13:50, jojimt wrote:

>> @@ -18,6 +39,7 @@
>>   - librbd-dev
>>   - lshw
>> when: ansible_os_family == "Debian"
>> +  ignore_errors: yes
>
> I don't like to ignore errors. Unfortunately, this task intermittently 
> returns apt failures. If anyone knows a fix/workaround for that, we 
> can add that and remove the ignore errors.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/ansible/pull/3/files#r46776650